src/main/scala/com/snowplowanalytics/piinguin/relay/PiinguinRelay.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2018 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+
+package com.snowplowanalytics.piinguin.relay
+
+// Scala
+import scala.collection.JavaConverters._
+import scala.concurrent.duration._
+import scala.concurrent.{ Await, Future }
+
+// Json4s
+import org.json4s._
+import org.json4s.JsonDSL._
+import org.json4s.MappingException
+
+// Kinesis
+import com.amazonaws.services.lambda.runtime.events.KinesisEvent
+
+// Piinguin Client
+import com.snowplowanalytics.piinguin.client.PiinguinClient
+
+// Analytics SDK
+import com.snowplowanalytics.snowplow.analytics.scalasdk.json.EventTransformer
+
+class PiinguinRelay {
```

Comment:
Is there any reasons why this is class and not an object?

Reply:
Following the snowplow standard https://github.com/snowplow/aws-lambda-scala-example-project/blob/master/src/main/scala/com.snowplowanalytics.awslambda/LambdaFunction.scala

Reply:
Not a standard, but rather historical artifact. I definitely can believe that this some kind of AWS Lambda limitation, but if not - should be simplified.